configurations/s02_pipeline_configuration.py

```diff
@@ -38,7 +38,14 @@
                     FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "gender", "gender"),
                     FlowResultConfiguration("wusc_leap_s02_kalobeyei_demogs", "disability", "disability"),
 
-                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e01", "leap_s02e01")
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e01", "leap_s02e01"),
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e02", "leap_s02e02"),
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e03", "leap_s02e03"),
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e04", "leap_s02e04"),
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e05", "leap_s02e05"),
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e06", "leap_s02e06"),
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e07", "leap_s02e07"),
+                    FlowResultConfiguration("wusc_leap_s02e01_kalobeyei_activation", "rqa_s02e08", "leap_s02e08")
```

Comment:
flow names

Reply:
Thanks for catching this, have updated it.